app/containers/HomeContainer/index.js

```diff
@@ -171,10 +171,10 @@ HomeContainer.defaultProps = {
 };
 
 const mapStateToProps = createStructuredSelector({
-  homeContainer: selectHomeContainer(),
-  reposData: selectReposData(),
-  reposError: selectReposError(),
-  repoName: selectRepoName()
+  homeContainer: selectHomeContainer,
+  reposData: selectSearchData(),
```

Comment:
let's rename the keys properly

Reply:
mapDispatchToProps will also have to change

Reply:
you'll have to change the name of the keys as well as wire the correct values

Reply:
and you can remove the portion responsible for rendering components based on the response of the API in this PR

Reply:
👍🏼